diff -ur compiler-rt-14.0.6.src/cmake/base-config-ix.cmake compiler-rt-14.0.6.src.patched/cmake/base-config-ix.cmake
--- compiler-rt-14.0.6.src/cmake/base-config-ix.cmake	2022-06-22 09:46:24.000000000 -0700
+++ compiler-rt-14.0.6.src.patched/cmake/base-config-ix.cmake	2022-09-05 15:21:58.852000000 -0700
@@ -120,24 +120,6 @@
   "Path where compiler-rt data files should be installed.")
 
 if(APPLE)
-  # On Darwin if /usr/include/c++ doesn't exist, the user probably has Xcode but
-  # not the command line tools (or is using macOS 10.14 or newer). If this is
-  # the case, we need to find the OS X sysroot to pass to clang.
-  if(NOT EXISTS /usr/include/c++)
-    execute_process(COMMAND xcrun -sdk macosx --show-sdk-path
-       OUTPUT_VARIABLE OSX_SYSROOT
-       ERROR_QUIET
-       OUTPUT_STRIP_TRAILING_WHITESPACE)
-    if (NOT OSX_SYSROOT OR NOT EXISTS ${OSX_SYSROOT})
-      message(WARNING "Detected OSX_SYSROOT ${OSX_SYSROOT} does not exist")
-    else()
-      message(STATUS "Found OSX_SYSROOT: ${OSX_SYSROOT}")
-      set(OSX_SYSROOT_FLAG "-isysroot${OSX_SYSROOT}")
-    endif()
-  else()
-    set(OSX_SYSROOT_FLAG "")
-  endif()
-
   option(COMPILER_RT_ENABLE_IOS "Enable building for iOS" On)
   option(COMPILER_RT_ENABLE_WATCHOS "Enable building for watchOS - Experimental" Off)
   option(COMPILER_RT_ENABLE_TVOS "Enable building for tvOS - Experimental" Off)
diff -ur compiler-rt-14.0.6.src/cmake/Modules/AddCompilerRT.cmake compiler-rt-14.0.6.src.patched/cmake/Modules/AddCompilerRT.cmake
--- compiler-rt-14.0.6.src/cmake/Modules/AddCompilerRT.cmake	2022-06-22 09:46:24.000000000 -0700
+++ compiler-rt-14.0.6.src.patched/cmake/Modules/AddCompilerRT.cmake	2022-09-05 15:17:27.524000000 -0700
@@ -372,7 +372,7 @@
         set_target_properties(${libname} PROPERTIES IMPORT_PREFIX "")
         set_target_properties(${libname} PROPERTIES IMPORT_SUFFIX ".lib")
       endif()
-      if(APPLE)
+      if(FALSE)
         # Ad-hoc sign the dylibs
         add_custom_command(TARGET ${libname}
           POST_BUILD  
