--- apple-libtapi-a66284251b46d591ee4a0cb4cf561b92a0c138d8/src/llvm/projects/libtapi/include/tapi/LinkerInterfaceFile.h
+++ apple-libtapi-fixed/src/llvm/projects/libtapi/include/tapi/LinkerInterfaceFile.h
@@ -78,7 +78,7 @@
   /// This value is temporary for bringup and will be removed.
   /// \since 1.4
   zippered = ~0U,
-} __attribute__((deprecated));
+};

 ///
 /// \brief Defines a list of Objective-C constraints.
