--- qtbase-everywhere-src-6.4.1-orig/src/plugins/platforms/xcb/CMakeLists.txt
+++ qtbase-everywhere-src-6.4.1/src/plugins/platforms/xcb/CMakeLists.txt
@@ -4,6 +4,8 @@
 ## XcbQpaPrivate Module:
 #####################################################################

+pkg_check_modules(X11 REQUIRED x11)
+
 qt_internal_add_module(XcbQpaPrivate
     CONFIG_MODULE_NAME xcb_qpa_lib
     INTERNAL_MODULE
@@ -40,6 +42,7 @@
         QT_NO_FOREACH
     INCLUDE_DIRECTORIES
         gl_integrations
+        ${X11_INCLUDE_DIRS}
     PUBLIC_LIBRARIES
         PkgConfig::XKB_COMMON_X11
         Qt::CorePrivate
