diff -ur qt-6.5.3-orig/src/corelib/thread/qfutureinterface.h qt-6.5.3/src/corelib/thread/qfutureinterface.h
--- qt-6.5.3-orig/src/corelib/thread/qfutureinterface.h	2024-02-04 20:42:39.951211354 -0800
+++ qt-6.5.3/src/corelib/thread/qfutureinterface.h	2024-05-27 14:34:55.782968046 -0700
@@ -452,7 +452,7 @@
 class QFutureInterface<void> : public QFutureInterfaceBase
 {
 public:
-    explicit QFutureInterface<void>(State initialState = NoState)
+    explicit QFutureInterface(State initialState = NoState)
         : QFutureInterfaceBase(initialState)
     { }
 
diff -ur qt-6.5.3-orig/src/gui/painting/qpaintengine_raster.cpp qt-6.5.3/src/gui/painting/qpaintengine_raster.cpp
--- qt-6.5.3-orig/src/gui/painting/qpaintengine_raster.cpp	2024-02-04 20:42:39.983211355 -0800
+++ qt-6.5.3/src/gui/painting/qpaintengine_raster.cpp	2024-05-27 14:32:02.759375639 -0700
@@ -3799,7 +3799,7 @@
         return;
 
     if (!m_clipLines)
-        m_clipLines = (ClipLine *)calloc(sizeof(ClipLine), clipSpanHeight);
+        m_clipLines = (ClipLine *)calloc(clipSpanHeight, sizeof(ClipLine));
 
     Q_CHECK_PTR(m_clipLines);
     QT_TRY {
