--- qtbase-everywhere-src-6.4.1-orig/src/gui/text/qplatformfontdatabase.cpp
+++ qtbase-everywhere-src-6.4.1/src/gui/text/qplatformfontdatabase.cpp
@@ -343,7 +343,7 @@
 {
     QString fontpath = QString::fromLocal8Bit(qgetenv("QT_QPA_FONTDIR"));
     if (fontpath.isEmpty())
-        fontpath = QLibraryInfo::path(QLibraryInfo::LibrariesPath) + "/fonts"_L1;
+        fontpath = QCoreApplication::applicationDirPath();

     return fontpath;
 }
